profile_library/shelly/Shelly 3EM/model.json

```
@@ -8,11 +8,14 @@
     "power_sensor_naming": "{} Device Power",
     "energy_sensor_naming": "{} Device Energy"
   },
-  "device_type": "smart_switch",
+  "device_type": "energy_meter",
```

```suggestion
  "device_type": "power_meter",
```

Needs to be one of the supported device_types.
`energy_meter` is not one of them.